File: 3RD SEMESTER/DS/Data Structure/palindrome.cpp
#include<iostream>
#include<string.h>

using namespace std;

bool recursive(char *p,int start,int end);
int main()
{
	char array[10];
	int length;
	cout << "Enter string : ";
	cin.getline(array,10);
	length=strlen(array);
	cout << "length : "<<length <<endl;
	int lastindex=length-1;
	if(recursive(array,0,lastindex))
	{
		cout << array << " is palindrome";
	}
	else
	{
		cout << array << " is not palindrome";
	}

}
bool recursive(char *p,int start,int end)
{
	if(start<end)
	{
		if(p[start]==p[end])
		{
			recursive(p,start+1,end-1);	
			
		}
		else
		{
			return false;
		}
	}
	
	
}

File: 6TH SEMESTER/Web Engineering - Lab/K20-1052/Task3/lab8/palindrome.php
<?php

$string = "madam";




function palindrome($string)
{

  for ($i = 0; $i < strlen($string) / 2; $i++) {
    if ($string[$i] != $string[strlen($string) - $i - 1]) {
      echo "not a palindrome";
      return;
    }
  }


  return $string . " is palindrome";
}




echo palindrome($string);
